Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -613,7 +610,7 @@ private void schrittPerParameterLaden() throws DAOException {
              * nachholen, damit die Liste vollstÃ¤ndig ist
              */
             if (this.page == null &&  getUser() != null) {
-                redirectToList();
+                // TODO: check if we still need this!
```

Comment:
- If you removing this method call then no different code is executed. Maybe you should first investigate if this method call is still needed and if not then remove it.
- In previous versions of this code the list of tasks / steps was refreshed (old and already removed `filterAll()` method) and a pagination object got new data. I don't know which error or not-refreshment was solved in the past.

Reply:
This method call is unnecessary because `redirectToList` did not perform any internal program logic but instead only returned a string, which here obviously wasn't used - because at this place we only had a `redirectToList()` instead of `return redirectToList()` in other places. This method call to `redirectToList` was probably never working to begin with.